package.json

```diff
@@ -1,8 +1,16 @@
 {
+  "name": "subnet-assets",
+    "version": "1.0.0",
+    "description": "repo containing info for subnets in the Avalanche ecosystem",
+    "repository": "git@github.com:ava-labs/subnet-assets.git",
+    "author": "Dov Brand <dov.brand@avalabs.org>",
+    "license": "MIT",
+    "type": "module",
   "scripts": {
     "format": "node format.mjs"
   },
   "devDependencies": {
-    "@ethersproject/address": "5.6.0"
+    "@ethersproject/address": "5.6.0",
+    "@types/node": "^17.0.31"
```

Comment:
I'd be interested in making this a typescript repo and run the files via https://github.com/lukeed/tsm#usage

Reply:
What would that do?

Reply:
All our .mjs or .js scripts could be .ts files since tsm automagically runs them via ts-node

Reply:
what does that require besides installing the dependency?

Reply:
Done